site-src/guides/metrics.md

```diff
@@ -1,109 +0,0 @@
-# Metrics
-
-This guide describes the current state of exposed metrics and how to scrape them.
-
-## Requirements
-
-To have response metrics, set the body mode to `Buffered` or `Streamed`:
-```
-apiVersion: gateway.envoyproxy.io/v1alpha1
-kind: EnvoyExtensionPolicy
-metadata:
-  name: ext-proc-policy
-  namespace: default
-spec:
-  extProc:
-    - backendRefs:
-      - group: ""
-        kind: Service
-        name: inference-gateway-ext-proc
-        port: 9002
-      processingMode:
-        request:
-          body: Buffered
-        response:
-          body: Buffered
-```
-
-If you want to include usage metrics for vLLM model server streaming request, send the request with `include_usage`:
-
-```
-curl -i ${IP}:${PORT}/v1/completions -H 'Content-Type: application/json' -d '{
-"model": "food-review",
-"prompt": "whats your fav movie?",
-"max_tokens": 10,
-"temperature": 0,
-"stream": true,
-"stream_options": {"include_usage": "true"}
-}'
-```
-
-## Exposed metrics
```

Comment:
should we keep the list of metrics somewhere?

Reply:
Updated, thanks!